voicevox_engine/tts_pipeline/katakana_english.py

```diff
@@ -0,0 +1,122 @@
+"""アルファベット読み上げ処理"""
```

Comment:
読み上げと言うと音声なのと、アルファベットと言うと文字っぽいので、こんな感じとか？
```suggestion
"""英単語をカタカナ読みにする処理"""
```